_articles/open-firmware-systems.md

```diff
@@ -33,10 +33,12 @@ Here is a [link](https://github.com/system76/ec) to the source code for our Open
 
 | System | Open Firmware | Open EC |
 |:------:|:-------------:|:-------:| 
+| addw2  | Yes           |         |
```

Comment:
addw2 has Open EC

Reply:
I fixed that in this commit: https://github.com/system76/docs/pull/539/commits/0bf9efbbd6a5ebda153ce5e83332f62d901abde6